airflow/migrations/versions/2c6edca13270_resource_based_permissions.py

```diff
@@ -307,7 +309,10 @@ def remap_permissions():
 
 def upgrade():
     """Apply Resource based permissions."""
+    log = logging.getLogger()
+    handlers = log.handlers[:]
```

Comment:
```suggestion
    handlers = log.handlers
```

Reply:
Won't it get mutated? I wanted to make a deep copy to avoid situation that we had some time ago in Airflow with handlers 😄

Reply:
`airflow db upgrade` will run in a separate process so I don't think it might cause any issue.

Reply:
However, I am fine with this too -- doesn't hurt 😉